doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -81,26 +87,26 @@ This subsection contains the parameters related to the resolved CFD-DEM around p
 
 * The ``ib particles pvd file`` parameter is the file's name that will be created to animate the particles. This file stores all the variables calculated for each of the particles. This file is compatible with Paraview.
 
-To sharpen the immersed boundary of each particles, a layer of cells around the immersed boundary can be refined forming a hypershell of refined cells.
+To sharpen the immersed boundary of each particle, a layer of cells around the immersed boundary can be refined forming a near-particle zone of refined cells. An effective radius, dependent on the shape, is calculated at the shape initialization. Its definition is chosen so that the refinement zone is close to the surface. For example: the torus' effective radius is the ring thickness. For the rectangle (or box), it is the average of the half-lengths. For a sphere, it is the actual radius.
```

Comment:
```suggestion
To sharpen the immersed boundary of each particle, a layer of cells around the immersed boundary can be refined forming a near-particle zone of refined cells. An effective radius, dependent on the shape, is calculated at the shape initialization. Its definition is chosen so that the refinement zone is close to the surface. For example: the torus' effective radius is the ring thickness. For the rectangle (or box), it is the average of the half-lengths. For a sphere, it is the radius of the sphere.
```
Good thinking here. I think it will be relevant to add the effective definition of the effective radius for each shapes where you define the shapes.